save-demo-agent/src/nativeMain/kotlin/com/saveourtool/save/demo/agent/utils/SetupUtils.kt

```diff
@@ -32,11 +33,9 @@ suspend fun setupEnvironment(demoUrl: String, setupShTimeoutMillis: Long, demoCo
     try {
         downloadDemoFiles(demoUrl, demoConfiguration)
     } catch (e: IllegalStateException) {
-        logError("Error while downloading files to agent: ${e.message ?: e.toString()}.")
+        logError("Error while downloading files to agent: ${e.describe()}.")
         throw e
-    }
-
-    // todo: chmod all the files
+    }.unzip()
```

Comment:
not really understand, all files are `zip` arhives?

Reply:
oh, okay we download them as zip, ok

Reply:
In `save-demo-agent` all files are sent in zip-archive, yes